cfgov/unprocessed/css/organisms/video-player.less

```diff
@@ -308,7 +313,12 @@ button.o-video-player_play-btn,
         display: block;
         // Pushes button out of masked window when not playing.
         margin-top: @standard-btn-height__px;
+
+        &:focus {
+        outline: 1px dotted #0072ce !important;
```

Comment:
For clarity swap out `#0072ce` for `@pacific`. We don't advertise it well but all our official CFPB colors have [Less variables](https://github.com/cfpb/design-system/blob/main/packages/cfpb-core/src/brand-colors.less#L69).

Reply:
Fixed!